.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    presets: ["es2015", "react"]
+}
```

Comment:
Yeah I originally had a babelrc file and then I wondered if I should bother since I'm not using babel-cli in this project